odbc/src/api/functions.rs

```diff
@@ -1345,7 +1346,16 @@ pub unsafe extern "C" fn SQLForeignKeys(
     _fk_table_name: *const Char,
     _fk_table_name_length: SmallInt,
 ) -> SqlReturn {
-    unsupported_function(MongoHandleRef::from(statement_handle), "SQLForeignKeys")
+    panic_safe_exec!(
```

Comment:
There is a unit test for Empty statement, but I don't see any test for SQLForeignKeys and SQLPrimaryKeys.
Could you please add them in functions.yaml? I realized that right now only SQLTablesW is tested and I'd rather add them now then later. Plus this increases the odd of finding missing metadata columns or properties reported incorrectly.

Reply:
At the moment `SQLForeignKeys` can be added to be tested in functions.yaml.  For some reason SQLForeignKeys is implented in `odbc-sys` but not `SQLPrimaryKeys`.  
We did have [SQL-1015](https://jira.mongodb.org/browse/SQL-1015) to investigate how to test these missing functions from the odbc-sys crate.

Reply:
Yea, I'm not sure about adding a test for PrimaryKeys then, since all it would be doing is ensuring that the metadata matches what I put in.

Reply:
Got it!
It is always good to check that we are returning an empty resultset and the metadata correctly though. It checks that we don't fail on initializing them. 
ah, odbc-sys is not making our life easy :'(